app/javascript/react/components/Map/Map.tsx

```diff
@@ -79,6 +80,8 @@ const Map = () => {
   const navigate = useNavigate();
   const { t } = useTranslation();
 
+  const isMobile1 = useMobileDetection(screenSizes.smallDesktop);
+
```

Comment:
I think it's not used anywhere, we can get rid of it, unless you had an idea what to do with it?

Reply:
Sorry about that, this is leftover.